bindings/matrix-sdk-ffi/src/api.udl

```diff
@@ -42,7 +42,7 @@ interface Client {
 
     string homeserver();
 
-    void start_sync();
+    void start_sync(u64? limit);
```

Comment:
I would just use an u32 or even smaller type here. The limit is clamped at the server side to 5000 anyways AFAIK.